app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -21,7 +21,7 @@ def index
 
       # Eager load / prevent Bullet::Notification::UnoptimizedQueryError
       with_includes = ApplicationChoice.includes(
-        %i[application_form provider offered_course_option site accredited_provider],
+        %i[application_form provider current_option current_course site accredited_provider],
```

Comment:
I wonder why this changed? Because of the move to `has_one` maybe?

Reply:
Probably, the new association declarations make Rails more aware of the relationships.